### ISSUE TITLE
helloSystem icons [UPDATED]

### ISSUE THREAD

**Issue by Anvith00** [helloSystem/hello#403]
Here are the updated icons for "About this computer" for helloSystem, in both png and svg formats. Do let me know if there's anything wrong with the icons.

![computer](https://user-images.githubusercontent.com/24383877/202855209-364d2a93-89b6-4383-a0da-4a4ce8b41813.png)
![computer](https://user-images.githubusercontent.com/24383877/202855204-cd0d40db-ffd4-4d78-a6ea-ec4c6b95feb5.svg)
![computer-laptop](https://user-images.githubusercontent.com/24383877/202855200-1eeaddf9-8533-4715-945f-9fd39f151619.png)
![computer-laptop](https://user-images.githubusercontent.com/24383877/202855186-97af2f6b-dfa4-4dda-93be-3d5ca5f08c8c.svg)

**Comment by probonopd** [comment]
Hello @Anvith00, thank you very much.

Could you please make a 128x128 px png for `computer-laptop.png`? You can check with:

```
Users-Laptop% file 'computer-laptop.png' 
computer-laptop.png: PNG image data, 128 x 99, 8-bit/color RGBA, non-interlaced
```

**Comment by Anvith00** [comment]
Yep, missed that by accident :sweat:. Thank you for letting me know.

[computer-laptop.zip](https://github.com/helloSystem/hello/files/10047712/computer-laptop.zip)
